### DISCUSSION TITLE
Additional Formatting Options

### DISCUSSION THREAD

**Discussion by WorldOfMaze** [kalkih/mini-graph-card#745] [Ideas]
I love the Mini Graph Card.  I just discovered it and will be using it in my dashboards.

There are a few features that would make the card a bit a more user friendly.  

1. Y-axis line
2. X- and Y-axis grid lines
3. X-axis line
4. X-axis labels

Thanks.

**Comment by Str00pwafel** [comment]
I would like to add on this idea. (I'm very new so if there is another way to do this excuse my noobness)

I would like to see a text-align option.

**Comment by dmshimself** [comment]
I've been trying to find a way to show the values on the X axis.  For example when grouping by hour to show the actual hour.  I know if I hover over a bar I can see the hour concerned then but like the History Graph, I'd like to be able to see this without having to hover.  Have I missed something here in configuring or is this a new feature request?

**Comment by KaherdinTristan** [comment]
+1 for this

**Comment by vaemarr** [comment]
+1 for this too. I'd like to glance at the chart without having to hover. At the moment, it's not very mobile/companion app friendly.

**Comment by Raven-Luminari** [comment]
+1 for X-Axis values. I'd like to see the day or date on the X-axis when showing 7-day history
